js/nodes/processes/interval.js

```diff
@@ -0,0 +1,21 @@
+
+function Interval(){
+    this.addOutput("", "process_array");
+    this.widget_1 = this.addWidget("number","Initial", 0, function(v){}, {});
+    this.widget_2 = this.addWidget("text","Final", "End", function(v){}, {});
+
+
+};
+
+Interval.title = "Interval";
+Interval.desc = "Time interval";
+
+Interval.prototype.onExecute = function() {
+    this.setOutputData(0, [this.widget_1.value, this.widget_2.value])
+};
+
+LiteGraph.registerNodeType("processes/interval", Interval);
```

Comment:
Move this to a separate category like ´input/interval´, process is reserved for Kratos processes

Reply:
Done